Api/Data/Countries/NO/PE/Elections.csv

```diff
@@ -11,11 +11,11 @@
 1985;Sainte Laguës (modified);1.4;0.0;-1;157;0
 1981;Sainte Laguës (modified);1.4;0.0;-1;155;0
 1977;Sainte Laguës (modified);1.4;0.0;-1;155;0
-1973;Sainte Laguës (modified);1.4;0.0;-1;156;0
+1973;Sainte Laguës (modified);1.4;0.0;-1;155;0
 1969;Sainte Laguës (modified);1.4;0.0;-1;150;0
 1965;Sainte Laguës (modified);1.4;0.0;-1;150;0
 1961;Sainte Laguës (modified);1.4;0.0;-1;150;0
 1957;Sainte Laguës (modified);1.4;0.0;-1;150;0
 1953;Sainte Laguës (modified);1.4;0.0;-1;150;0
-1949;d'Hondt;1.0;0.0;-1;112;0
-1945;d'Hondt;1.0;0.0;-1;112;0
\ No newline at end of file
+1949;d'Hondt;1.0;0.0;-1;150;0
```

Comment:
Did we confirm this with Bernt/some source? It seems like an odd "typo" type of data error

Reply:
Yes, he flagget the errors with 112 seats, and the 156 one indirectly. It seems the new values are correct when comparing to Celius.